syntax = "proto3";
package pulse;
option java_package = "com.kitware.physiology.pulse";

import "cdm/ElectroCardioGram.proto";

message AnesthesiaMachineData
{
  bool                                         Inhaling                       = 1;
  double                                       CurrentBreathingCycleTime_s    = 2;
  double                                       InspirationTime_s              = 3;
  double                                       OxygenInletVolumeFraction      = 4;
  double                                       TotalBreathingCycleTime_s      = 5;
}

message ElectroCardioGramData
{
  double                                       HeartRythmTime_s               = 1;
  double                                       HeartRythmPeriod_s             = 2;
  cdm.ElectroCardioGramWaveformListData        Waveforms                      = 3;
}


message InhalerData
{
  
}